browser/typescript/v2/react-app/src/App.tsx

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+import { DefaultOptions, EventWithOptionalProperties, ampli } from './ampli';
+
+const { REACT_APP_AMPLITUDE_API_KEY = '', REACT_APP_SEGMENT_WRITE_KEY = '' } = process.env;
+
+const userId = 'ampli-v2-browser-ts-user-id';
+
+/**
+ * Start by calling ampli.load()
+ *
+ * 'ampli' is the default instance of Ampli()
+ */
+
+/**
+ * When you pull your tracking plan you can use the defaults and call load() without arguments
+ *
+ * This requires connecting your account via `ampli pull`
+ * which will set you API key in the generated Ampli SDK
+ */
+// ampli.load();
+
+/**
+ * OR Specify an environment
+ */
+// ampli.load({ environment: 'development' })
+
+/** OR Provide a specific Amplitude API key */
+// ampli.load({ client: { apiKey: REACT_APP_AMPLITUDE_API_KEY } })
+
+/**
+ * OR Use an existing Amplitude instance
+ * requires "import amplitude from '@amplitude/analytics-browser';"
+ */
+// amplitude.init(REACT_APP_AMPLITUDE_API_KEY, undefined, { ...DefaultOptions, logLevel: 3 });
+// ampli.load({ client: { instance: amplitude } });
+
+/**
+ * OR Use an existing window.amplitude instance from code snippet (not recommended)
+ */
+// window.amplitude.init(REACT_APP_AMPLITUDE_API_KEY);
+// ampli.load({ client: { instance: window.amplitude as any } });
+
+/**
+ * OR Specify AmplitudeClient 'options'
+ */
+ampli.load({
+  client: {
+    apiKey: REACT_APP_AMPLITUDE_API_KEY,
+    options: { ...DefaultOptions, logLevel: 3 },
```

Comment:
configurations: { ...DefaultConfiguration, logLevel: 3 },

Reply:
This does not match the index.ts. Please make sure to run the ampli example and check the events in the event stream before merging into the main branch.